bin/source_of_truth/commands_source_of_truth.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Sharezone UG (haftungsbeschränkt)
+# Licensed under the EUPL-1.2-or-later.
+#
+# You may obtain a copy of the Licence at:
+# https://joinup.ec.europa.eu/software/page/eupl
+#
+# SPDX-License-Identifier: EUPL-1.2
+
+# This file is the "source of truth" for running commands that are the same for all operating systems.
+# We want to run the same command everywhere, on local dev machines and on CI systems.
+# Since Sharezone can be developed on Linux / Mac / Windows just putting the command in a a .sh file isn't enough, since Windows couldn't execute it.
+# 
+# CI:
+# When running these commands via CI we just read them in inside the bash script and execute the command.
+# This saves us first installing Dart for e.g. just executing a simple go script.
```

Comment:
Just a site node: This would be a advantage of [sidekick](https://github.com/phntmxyz/sidekick) that we could run our CLI directly without Dart and we wouldn't the need have this setup of `commands_source_of_truth.yaml`, etc.

Reply:
> Executing the CLI requires no extra dependencies. The entrypoint can be executed as shell script. That makes it easy to use on CI/CD servers.
>
> By calling the entrypoint shell script, it automatically downloads a (pinned) Dart runtime and compiles the CLI project.

I mean yes and no.  
You can just call it and Dart will be installed automatically, that's not the issue though.  
What I wanted to do here was that we don't have to install Dart at all.
...at least for now - in the future it might be smart to create a docker image for our CI/CD which has all dependencies installed already and where we can just call the Dart commands.

Reply:
To be perfectly honest since everything takes so long anyways it might just be better to still use Dart anyways even if that adds the install time 😅
Kinda over engineered this one I guess 😬

Reply:
Yeah, especially Dart only things are still pretty fast. Installation time is only ~ 10 seconds. Only Flutter things suck because they have a overhead of 60 - 90 seconds.

Reply:
I will merge this anyways for now, because (that was also partly my reasoning) I like that the license check is very fast, so that there is direct feedback when pushing new commits.

Reply:
Ah didnt see your last comment:(